co.headsetapp.headset.yml

```diff
@@ -32,6 +33,9 @@ modules:
         dest-filename: headset.sh
         commands:
           - zypak-wrapper /app/headset/headset "$@"
+          - for i in {0..9}; do \
+          -    test -S $XDG_RUNTIME_DIR/discord-ipc-$i || ln -sf {app/com.discordapp.Discord,$XDG_RUNTIME_DIR}/discord-ipc-$i; \
+          -  done
```

Comment:
These lines should be placed before we run Headset, before `zypak-wrapper /app/headset/headset "$@"`

Reply:
This is true, completely forgot to swap them round again xD